compatLib/compatLibVS/src/main/java/app/lawnchair/compatlib/twelve/ActivityOptionsCompatVS.java

```diff
@@ -15,8 +15,9 @@ public ActivityOptions makeCustomAnimation(
             Context context,
             int enterResId,
             int exitResId,
-            Runnable callback,
-            Handler callbackHandler) {
+            Handler callbackHandler,
+            Runnable startedListener,
+            Runnable finishedListener) {
         return ActivityOptions.makeCustomTaskAnimation(
```

Comment:
I just wonder why use `makeCustomTaskAnimation` here instead of `makeCustomAnimation`?

Reply:
What do you mean?